### PR TITLE
Added CS Fixer, corrected code for compliance with PER 2.0

### LINKED ISSUE #56: Add CS checker and fixer
Add CS checker and fixer that will:
- [x] Check the code for compliance with PER-2 https://github.com/buggregator/trap/pull/67
- [x] #68
- Take into account some custom aspects:
  - We use FQN for functions in the global namespace
  - We use FQN for classes in the global namespace
  
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by chachanagov**
## What was changed

Сode is brought into compliance with PER-2.
Added action for automatic code checking during Pull Requests

## Checklist

- Closes #56

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -39,4 +39,3 @@ function trap(mixed ...$values): TrapHandle
     /** @psalm-suppress MixedAssignment */
     AbstractCloner::$defaultCasters[EnumValue::class] ??= [ProtobufCaster::class, 'castEnum'];
 }
-
```
Comment (roxblnfk) [inline src/functions.php:42]: Why it was removed? o_O

File: src/Sender/Console/Renderer/VarDumper.php [hunk 2]
```diff
@@ -51,11 +51,10 @@ public function render(OutputInterface $output, Frame $frame): void
 
     private function getDescriber(): DumpDescriptorInterface
     {
-        return new class() implements DumpDescriptorInterface {
+        return new class () implements DumpDescriptorInterface {
```
[the remark this answers is not in the document]
Comment (roxblnfk) [reply src/Sender/Console/Renderer/VarDumper.php:54]: Is there any option to fix it?

File: src/functions.php (hunk 1 above)
Comment (chachanagov) [reply src/functions.php:42]: There were two line breaks, one extra.

File: src/Test/Proto/Message.php
[the remark this answers is not in the document]
Comment (chachanagov) [reply src/Test/Proto/Message.php:1]: Added directory to exclude. And revert commit.

File: src/Sender/Console/Renderer/VarDumper.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (chachanagov) [reply src/Sender/Console/Renderer/VarDumper.php]: Added fix for anonymous classes.

File: src/Test/Proto/Message.php
[the remark this answers is not in the document]
Comment (roxblnfk) [reply src/Test/Proto/Message.php:1]: One file still has changes
image

**Review by roxblnfk — APPROVED** [review]
LGTM

**Pull request merged**